inference-engine/src/gna_plugin/transformations/reoder_activation_and_pooling.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
Typo in the name of the file: reoder->reorder.

Reply:
thanks, fixed it